engine_comparison/dispatcher.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+. $(dirname $0)/../common.sh
+
+build_engine() {
+  # [[ ! -e ~/$FENGINE_CONFIGS_DIR/$1 ]] && echo "cant do" && exit 1
+  echo "Building $FUZZING_ENGINE"
+  # Probably use another script, to easily use env vars
+  #
+  # if libfuzzer
+  # if afl
+}
+
+build_benchmark_using() {
+  BENCHMARK=$1
+  FENGINE_CONFIG=$2
+  # . $FENGINE_CONFIG
+
+  BUILDING_DIR=RUN-${THIS_BENCHMARK}
+  rm -rf $BUILDING_DIR
+  mkdir $BUILDING_DIR
+  cd $BUILDING_DIR
+  echo "Filling $BUILDING_DIR"
+
+  # [[ ! -e ~/$FTS/$1/build.sh ]] && echo "cant build" && exit 1
+  ~/$FTS/$1/build.sh $FENGINE_NAME $FUZZING_ENGINE
```

Comment:
$FTS is a variable I'd like to pass from `begin_experiment`, which will scp it's own local copy of FTS